docs/docs/v3.1.0-migration.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
We seem to only have migration docs for the major versions, but not for the minor. I added this for that reason. Not sure if there is a better way to go about this.